src/synthesizer/emission_models/operations.py

```diff
@@ -53,6 +55,9 @@ def __init__(self, grid, extract, fesc):
         # Attach the escape fraction
         self._fesc = fesc
 
+        # Attach the wavelength mask
+        self._lam_mask = lam_mask
```

Comment:
Following from comment above, probably better to attach this at the base model if we do allow all operations to use a wavelength mask.